homeassistant/components/influxdb/sensor.py

```diff
@@ -174,13 +225,28 @@ def setup_platform(
                 entities.append(InfluxSensor(hass, influx, query))
             else:
                 _LOGGER.error(NO_BUCKET_ERROR, query[CONF_BUCKET])
-    else:
+
+    if CONF_QUERIES in config:
```

Comment:
```suggestion
    elif CONF_QUERIES in config:
```

`CONF_QUERIES_FLUX` and `CONF_QUERIES` are exclusiv